pkg/agent/controller/reconciliation_test.go

```diff
@@ -156,4 +159,28 @@ var _ = Describe("Reconciliation", func() {
 			t.awaitNoEndpointSlice(t.cluster2.localEndpointSliceClient)
 		})
 	})
+
+	When("a local EndpointSlice with the old naming convention sans namespace exists on startup", func() {
```

Comment:
How about a test with two slices with the same name in different namespaces?

Reply:
So a service with the same name in 2 namespaces. I can look into that. But this test is still valid as is.

Reply:
Yes, I’m not saying this test is invalid; just asking for an additional test that verifies the behaviour in the bug.

Reply:
Added with https://github.com/submariner-io/lighthouse/pull/964/commits/3c961200cc28f9b68291cf4ec4e9a5aba642b075